docs/v1/details/tracking-llm-calls.mdx

```diff
@@ -3,8 +3,21 @@ title: "Tracking LLM Calls"
 description: "Tracking LLM Calls using the AgentOps SDK"
 ---
 
-When the AgentOps SDK detects the `openai` as an installable module, it will automatically
-start tracking OpenAI calls by instrumenting chat completions to track their usage.
+### How it works
+
+When the AgentOps SDK detects the `openai`, `litellm`, or `cohere` as installed modules, it will automatically
+start tracking their usage. No further work is required from you! 😊
+
+### Not working? 
+
+Try these steps:
+1. Make sure you have the latest version of the AgentOps SDK installed. We are constantly updating it to support new LLM libraries and releases.
+2. Make sure you are calling `agentops.init()` *after* importing the LLM module but *before* you are calling the LLM method.
+3. Make sure the `instrument_llm_calls` parameter of `agentops.init()` is set to `True`.
+
+Still not working? Please let us know! You can find us on [Discord](https://discord.gg/DR2abmETjZ),
+ [GitHub](https://github.com/AgentOps-AI/agentops), 
+ or email us at [shawn@agentops.ai](mailto:shawn@agentops.ai).
```

Comment:
should we have a eng@agentops.ai ?

Reply:
I can make one, but it will just go to my email